spec/factories/ticket_purchases.rb

```diff
@@ -4,5 +4,6 @@
     conference
     ticket
     quantity 10
+    paid false
```

Comment:
It is false as default in the database, so this shouldn't be needed, right? :thinking: